### ISSUE TITLE
[BUG] verifyer failed on GitHub action when testing problem with large data

### ISSUE THREAD

**Issue by Tiphereth-A** [competitive-verifier/competitive-verifier#162]
Log:

<details>

```
  INFO:competitive_verifier.oj.tools.download_command:141: download[Start]: https://judge.yosupo.jp/problem/convolution_mod_large into .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test
  INFO:competitive_verifier.oj.tools.download_command:143: download[Run]: https://judge.yosupo.jp/problem/convolution_mod_large
  DEBUG:competitive_verifier.oj.tools.utils:43: User-Agent: competitive-verifier/3.2.2 (+https://github.com/competitive-verifier/competitive-verifier)
  INFO:onlinejudge.utils:53: load cookie from: /home/runner/work/CP-lib/CP-lib/.competitive-verifier/cache/online-judge-tools/cookie.txt
  INFO:onlinejudge.service.library_checker:106: $ /usr/bin/python3 .competitive-verifier/cache/library-checker-problems/generate.py .competitive-verifier/cache/library-checker-problems/convolution/convolution_mod_large/info.toml
  08:00:44 [INFO] Start convolution_mod_large
  08:00:44 [INFO] generate params.h
  08:00:44 [INFO] compile checker
  08:00:48 [INFO] compile solution
  08:00:49 [INFO] compile generators
  08:00:49 [INFO] compile example.in
  08:00:49 [INFO] compile small.cpp
  08:00:49 [INFO] compile medium.cpp
  08:00:50 [INFO] compile random.cpp
  08:00:51 [INFO] compile max_random.cpp
  08:00:51 [INFO] compile fft_killer.cpp
  08:00:52 [INFO] compile medium_all_zero.cpp
  08:00:52 [INFO] compile max_ans_zero.cpp
  08:00:53 [INFO] compile signed_overflow.cpp
  08:00:54 [INFO] compile unsigned_overflow.cpp
  08:00:54 [INFO] compile medium_pre_suf_zero.cpp
  08:00:55 [INFO] compile small_and_large.cpp
  08:00:55 [INFO] clear input .competitive-verifier/cache/library-checker-problems/convolution/convolution_mod_large/in
  08:00:55 [INFO] gen example.in 2cases
  08:00:55 [INFO] gen small.cpp 16cases
  08:00:55 [INFO] gen medium.cpp 3cases
  08:00:55 [INFO] gen random.cpp 3cases
  08:00:57 [INFO] gen max_random.cpp 2cases
  08:00:58 [INFO] gen fft_killer.cpp 10cases
  08:01:08 [INFO] gen medium_all_zero.cpp 3cases
  08:01:08 [INFO] gen max_ans_zero.cpp 1cases
  08:01:09 [INFO] gen signed_overflow.cpp 1cases
  08:01:09 [INFO] gen unsigned_overflow.cpp 1cases
  08:01:09 [INFO] gen medium_pre_suf_zero.cpp 5cases
  08:01:09 [INFO] gen small_and_large.cpp 4cases
  08:01:10 [INFO] clear output .competitive-verifier/cache/library-checker-problems/convolution/convolution_mod_large/out
  08:01:10 [INFO] ANS      1 msecs : example_00 : b''
  08:01:10 [INFO] ANS      1 msecs : example_01 : b''
  08:01:10 [INFO] ANS      1 msecs : small_00 : b''
  08:01:10 [INFO] ANS      1 msecs : small_01 : b''
  08:01:10 [INFO] ANS      1 msecs : small_02 : b''
  08:01:10 [INFO] ANS      1 msecs : small_03 : b''
  08:01:10 [INFO] ANS      1 msecs : small_04 : b''
  08:01:10 [INFO] ANS      1 msecs : small_05 : b''
  08:01:10 [INFO] ANS      1 msecs : small_06 : b''
  08:01:10 [INFO] ANS      1 msecs : small_07 : b''
  08:01:10 [INFO] ANS      1 msecs : small_08 : b''
  08:01:10 [INFO] ANS      1 msecs : small_09 : b''
  08:01:10 [INFO] ANS      1 msecs : small_10 : b''
  08:01:10 [INFO] ANS      1 msecs : small_11 : b''
  08:01:10 [INFO] ANS      1 msecs : small_12 : b''
  08:01:10 [INFO] ANS      1 msecs : small_13 : b''
  08:01:10 [INFO] ANS      1 msecs : small_14 : b''
  08:01:10 [INFO] ANS      1 msecs : small_15 : b''
  08:01:10 [INFO] ANS      3 msecs : medium_00 : b''
  08:01:10 [INFO] ANS      2 msecs : medium_01 : b''
  08:01:10 [INFO] ANS      2 msecs : medium_02 : b''
  08:01:12 [INFO] ANS   1911 msecs : random_00 : b''
  08:01:14 [INFO] ANS   2448 msecs : random_01 : b''
  08:01:18 [INFO] ANS   3671 msecs : random_02 : b''
  08:01:23 [INFO] ANS   4399 msecs : max_random_00 : b''
  08:01:27 [INFO] ANS   4417 msecs : max_random_01 : b''
  08:01:31 [INFO] ANS   4377 msecs : fft_killer_00 : b''
  08:01:36 [INFO] ANS   4403 msecs : fft_killer_01 : b''
  08:01:40 [INFO] ANS   4368 msecs : fft_killer_02 : b''
  08:01:45 [INFO] ANS   4381 msecs : fft_killer_03 : b''
  08:01:49 [INFO] ANS   4389 msecs : fft_killer_04 : b''
  08:01:53 [INFO] ANS   4416 msecs : fft_killer_05 : b''
  08:01:58 [INFO] ANS   4376 msecs : fft_killer_06 : b''
  08:02:02 [INFO] ANS   4375 msecs : fft_killer_07 : b''
  08:02:06 [INFO] ANS   4373 msecs : fft_killer_08 : b''
  08:02:11 [INFO] ANS   4381 msecs : fft_killer_09 : b''
  08:02:11 [INFO] ANS      2 msecs : medium_all_zero_00 : b''
  08:02:11 [INFO] ANS      2 msecs : medium_all_zero_01 : b''
  08:02:11 [INFO] ANS      2 msecs : medium_all_zero_02 : b''
  08:02:15 [INFO] ANS   4402 msecs : max_ans_zero_00 : b''
  08:02:15 [INFO] ANS      1 msecs : signed_overflow_00 : b''
  08:02:15 [INFO] ANS      1 msecs : unsigned_overflow_00 : b''
  08:02:15 [INFO] ANS      1 msecs : medium_pre_suf_zero_00 : b''
  08:02:15 [INFO] ANS      1 msecs : medium_pre_suf_zero_01 : b''
  08:02:15 [INFO] ANS      1 msecs : medium_pre_suf_zero_02 : b''
  08:02:15 [INFO] ANS      1 msecs : medium_pre_suf_zero_03 : b''
  08:02:15 [INFO] ANS      1 msecs : medium_pre_suf_zero_04 : b''
  08:02:18 [INFO] ANS   2417 msecs : small_and_large_00 : b''
  08:02:20 [INFO] ANS   2441 msecs : small_and_large_01 : b''
  08:02:23 [INFO] ANS   2438 msecs : small_and_large_02 : b''
  08:02:25 [INFO] ANS   2421 msecs : small_and_large_03 : b''
  INFO:onlinejudge.utils:56: save cookie to: /home/runner/work/CP-lib/CP-lib/.competitive-verifier/cache/online-judge-tools/cookie.txt
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/example_00.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/example_00.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/example_01.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/example_01.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_00.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_00.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_01.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_01.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_02.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_02.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_03.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_03.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_04.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_04.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_05.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_05.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_06.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_06.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_07.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_07.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_08.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_08.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_09.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/fft_killer_09.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/max_ans_zero_00.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/max_ans_zero_00.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/max_random_00.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/max_random_00.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/max_random_01.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/max_random_01.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_00.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_00.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_01.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_01.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_02.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_02.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_all_zero_00.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_all_zero_00.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_all_zero_01.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_all_zero_01.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_all_zero_02.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_all_zero_02.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_00.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_00.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_01.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_01.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_02.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_02.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_03.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_03.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_04.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/medium_pre_suf_zero_04.out
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/random_00.in
  DEBUG:competitive_verifier.oj.tools.download_command:132: saved to: .competitive-verifier/cache/problems/e8be5d7faaf48a5884ce8f6590912a22/test/random_00.out
  ERROR:competitive_verifier.oj.tools.download_command:173: Failed to download: [Errno 28] No space left on device
  Traceback (most recent call last):
    File "/home/runner/.local/lib/python3.10/site-packages/competitive_verifier/oj/tools/download_command.py", line 154, in run_wrapper
      run(
    File "/home/runner/.local/lib/python3.10/site-packages/competitive_verifier/oj/tools/download_command.py", line 131, in run
      fh.write(data)
  OSError: [Errno 28] No space left on device
```

</details>

Maybe we need to remove some test data which have been used 🤔

**Comment by kzrnm** [comment]
Since the error is the huge test case files output by [yosupo06/library-checker-problems](https://github.com/yosupo06/library-checker-problems), I don't plan to handle this issue with competitive-verifier.

If you fix it and make a new pull request, I will consider it.

-----

I think you can solve the problem with split jobs.

```yaml
# pseudo actions
job_large:
- name: oj-resolve
      uses: competitive-verifier/actions/oj-resolve@v2
      with:
      output-path: verify_files.json
      exclude: |
            tests
- name: Verify
      uses: competitive-verifier/actions/verify@v2

job_other:
- name: oj-resolve
      uses: competitive-verifier/actions/oj-resolve@v2
      with:
      output-path: verify_files.json
      exclude: |
            tests_convolution_mod_large
- name: Verify
      uses: competitive-verifier/actions/verify@v2

check_job:
- name: Merge results
      uses: competitive-verifier/actions/merge-result@v2
```

**Comment by Tiphereth-A** [comment]
I have tried to split that test in <https://github.com/Tiphereth-A/CP-lib/actions/runs/12121522762>, and it seems that we can not fix this in either this repo or my CP-lib... sad.

**Comment by kzrnm** [comment]
Storage within the Action is guaranteed up to 14GB.

https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

**Comment by kzrnm** [comment]
The increased storage consumption was due to this pull request.


https://github.com/yosupo06/library-checker-problems/pull/1285

**Comment by Tiphereth-A** [comment]
FYI I resolved this issue by https://github.com/yosupo06/library-checker-problems/issues/1293#issuecomment-2543898075

**Comment by Tiphereth-A** [comment]
I wonder is https://github.com/yosupo06/library-checker-problems/issues/1293#issuecomment-2543891406 the main reason which causes storage limitation exceeded?

**Comment by kzrnm** [comment]
Your problem may be fixed in v3.3.0.

https://github.com/competitive-verifier/zzz_Tiphereth-A_CP-lib/actions/runs/12529018775
https://github.com/competitive-verifier/zzz_Tiphereth-A_CP-lib/actions/runs/12529093753

**Comment by kzrnm** [comment]
@Tiphereth-A 
Your problem may be fixed in v3.3.0.

https://github.com/competitive-verifier/zzz_Tiphereth-A_CP-lib/actions/runs/12529018775/attempts/1
https://github.com/competitive-verifier/zzz_Tiphereth-A_CP-lib/actions/runs/12529018775

**Comment by Tiphereth-A** [comment]
> @Tiphereth-A Your problem may be fixed in v3.3.0.
> 
> https://github.com/competitive-verifier/zzz_Tiphereth-A_CP-lib/actions/runs/12529018775/attempts/1 https://github.com/competitive-verifier/zzz_Tiphereth-A_CP-lib/actions/runs/12529018775

Thank you!
